index.html

```diff
@@ -25,22 +25,28 @@
           <nav class="menu-bars">
             <span class="material-symbols-outlined"> menu </span>
           </nav>
+          <nav class="toolbar">
+            <a href="#">Portfolio</a>
+            <a href="#">About</a>
+            <a href="#">Contact</a>
+          </nav>
```

Comment:
Good job! but as you know it's required to follow the Figma template, so please make sure to use the same color, sizing, and spacing for your navbar.

![image](https://user-images.githubusercontent.com/34779928/194420894-d6d8a75a-6456-4fd5-bca9-4007e19dde7f.png)